docs/content/integrations/dbt/reference.mdx

```diff
@@ -658,15 +658,16 @@ def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
 
 ---
 
-## Emit column-level metadata as materialization metadata <Experimental />
+## Emit column-level metadata as materialization metadata
 
 <Note>
   <strong>
     Emitting column-level metadata as materialization metadata is currently an
     experimental feature.{" "}
   </strong>{" "}
-  To use this feature, you'll need to be on at least `dagster>=1.6.12` and
-  `dagster-dbt>=0.22.12`.
+  To use this feature, you'll need to be on at least{" "}
+  <code>dagster>=1.6.12</code> and
+  <code>dagster-dbt>=0.22.12</code>.
```

Comment:
Since you're here already, mind changing this? A number of fixes landed between 1.6.12 and now, so just want to make sure folks are on the patch with all those fixes.

```suggestion
  To use this feature, you'll need to be on at least{" "}
  <code>dagster>=1.7.0</code> and
  <code>dagster-dbt>=0.23.0</code>.
```